samples/js/js-valueentity-shopping-cart/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "js-valueentity-shopping-cart",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
below is `kalix-javascript-sdk": "0.33.5"`, shouldn't we update that? and I would like to publish a new version that includes #279

Reply:
There's a script that updates all sample dependencies like this _after_ doing a release, (they are tested with current as part of PR validation however)

Reply:
> below is kalix-javascript-sdk": "0.33.5", shouldn't we update that?

ow, hadn't noticed that :/ - yes, definitely

> and I would like to publish a new version that includes https://github.com/lightbend/akkaserverless-javascript-sdk/pull/279

:+1:

Reply:
I'm not sure I understand. I'm fine with merging this and then cut a new release. Note that we have already published 1.0.0-M3 and next would be M4

Reply:
ah, gotcha, I didn't realize 0.33.5 is from before the renaming (because the artifact name *was* updated in the package.json here, confusing). No need to wait then :+1:

Reply:
Bumping samples in https://github.com/lightbend/akkaserverless-javascript-sdk/pull/282 (but that isn't blocking a release if you want to make one @patriknw )